src/odemis/driver/hamamatsurx.py

```diff
@@ -753,7 +913,10 @@ def _getStreakUnitTimeRangeChoices(self):
         choices = []
         for choice in choices_raw:
             choice_raw = choice.split(" ")
-            choices.append(self.parent.convertUnit2Time(choice_raw[0], choice_raw[1]))
+            if len(choice_raw) == 1:
+                choices.append(int(choice_raw[0]))
+            else:
+                choices.append(self.parent.convertUnit2Time(choice_raw[0], choice_raw[1]))
```

Comment:
Please explain that the len==1 means there is no unit.
=> Need to convert to the corresponding (real) time value afterwards.